src/pages/p2p/components/_dp2p.tsx

```diff
@@ -159,7 +159,7 @@ const DP2P = ({ P2P, reverse }: DP2PProps) => {
                 ></StyledIFrame>
 
                 {P2P.map((item, index) => {
-                    let is_even = reverse ? (index + 1) % 2 : index % 2
+                    const is_even = reverse ? (index + 1) % 2 : index % 2
                     return (
                         <Row flex_direction={!is_even ? 'row' : 'row-reverse'} key={index}>
```

Comment:
Let's make the check base don truthy value, more easier to read

Reply:
This is not related to my changes, just warning or error fix while starting project.